src/style.css

```diff
@@ -0,0 +1,116 @@
+.main {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+}
+
+.card__image {
+  align-self: center;
+  width: 160px;
+  height: 134px;
+  object-fit: contain;
+  margin: 0 0 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0 0 4px;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.card__code {
+  font-size: 10px;
+  font-weight: 400;
+  color: #616070;
+  margin: 0 0 16px;
```

Comment:
you just can write - `margin-bottom` in this case

Reply:
Wrote it this way, as I forgot to reset the default margins for the <p> tag. Changed it accordingly.